test/common_utils.py

```diff
@@ -933,30 +933,35 @@ def random_square_matrix_of_rank(l, rank):
 
 def random_symmetric_matrix(l, *batches):
     A = torch.randn(*(batches + (l, l)))
-    for i in range(l):
-        for j in range(i):
-            A[..., i, j] = A[..., j, i]
+    A = A + A.transpose(-2, -1)
```

Comment:
could you `.div_(2)` at the line above to keep the magnitude similar to before this change?